Orange/data/sql/backend/postgres.py

```diff
@@ -45,8 +46,9 @@ def _create_extensions(self):
                 query = "CREATE EXTENSION IF NOT EXISTS {}".format(ext)
                 with self.execute_sql_query(query):
                     pass
-            except OperationalError:
+            except BackendError:
                 warnings.warn("Database is missing extension {}".format(ext))
+                self.missing_extension = ext
```

Comment:
Are you sure you want to overwrite  `missing_extension` with ext in the for loop?
This will only save the last one missing... maybe missing_extensions could be a list and ext is appended here.